Gemfile

```diff
@@ -3,14 +3,14 @@
 source 'https://rubygems.org'
 
 group :development, :test do
-  gem 'gem_generator', '~> 0.3.0'
+  gem 'gem_generator', '~> 0.4.0'
```

Comment:
It would be great to support at least Ruby 2.7 while it's still officially maintained.
Do we actually need to bump this version up?
If we used `~> 0.3` instead, we should be able to reinstate Ruby 2.7 as well?
My understanding is that `bundle outdated` should pass on both 2.7 and 3.0+ if we do this?

Reply:
@iMacTia you're right, it's also an option. Not too good because of some potential fixes and new abilities, but it is.

I'll think also about downgrade some dependencies to allow these not too old Ruby versions, maybe additional conditions for `gorilla_patch` or something.

I just showed the current situation and wanted to know your opinion.